android/src/com/frostwire/android/util/ImageLoader.java

```diff
@@ -148,6 +149,30 @@ private ImageLoader(Context context) {
         picasso.setIndicatorsEnabled(false);
     }
 
+    public void loadAndTransform(final Uri primaryUri, final Uri secondaryUri, final Transformation transformation, final ImageView imageView, final boolean cache) {
```

Comment:
create an interface inside ImageLoader that mimics the `Transformation` from picasso. Feel free to put another name (like Filter) and then use that interface for the ImageLoader API.